astropy/cosmology/parameter/_core.py

```diff
@@ -100,6 +116,13 @@ class Parameter:
     if not PYTHON_LT_3_10:
         _: KW_ONLY
 
+    default: Any = MISSING
+    """Default value of the Parameter.
+
+    If ``MISSING`` (default), the Parameter must be set when initializing the
```

Comment:
Maybe
"""
By default, set to ``MISSING``, which indicates the parameter must be set
when initializing the cosmology.
"""

Reply:
Done!